onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -44,287 +45,326 @@ std::map<std::string, int> CountOpsInGraph(const Graph& graph) {
   }
   return op_to_count;
 }
+
+// Takes as input an onnx file uri and a list of transformers and rules. Then it creates,
+// initializes, and returns an inference session.
+std::unique_ptr<InferenceSession> CreateSession(const std::string& model_uri,
```

Comment:
>CreateSession [](start = 34, length = 13)

I think we should validate inference_session intiialization etc... in inference_session_test

Here we should only test each individual transformer....

My current PR I have added session initialization test in inference_session_test file

Reply:
Done.